### DISCUSSION TITLE
"Save" images shared by others in my timeline.

### DISCUSSION THREAD

**Discussion by vivek1052** [immich-app/immich#3843] [General]
Would be nice if I can save (not download) the pictures shared by others via shared album or partner sharing so that it appears in my timeline. 

I guess we only need to link the asset to my owner id. Its actual location can remain as it is.

**Comment by AngelaDMerkel** [comment]
It would also be nice if it identified who shared the image too, including when images are added by guests in shared albums. This is one of the most useful and powerful features offered by GPhotos and is very helpful in organisation.

**Comment by lippertmarkus** [comment]
This is the most important feature to me that I'm currently missing. 

Partner Sharing doesn't work for me as me and my family still want to have everything private and decide via shared albums what to share with other family members. 

As the shared albums usually contains photos from activites done together I also want to see those photos in my timeline (/photos). I would love to see the equivalent of "Save photos" that's available shared albums in Google Photos:
<img width="299" alt="image" src="https://github.com/immich-app/immich/assets/1463140/ebf4bdcc-f43a-44ec-8ff2-3c18b3b53a4f">

**Comment by sierraalpha83** [comment]
This feature would be really great!

Same use case as @lippertmarkus for me.

**Comment by vishwa5854** [comment]
Any updates on this feature, any timeline on when it might come live ?
Btw, thanks for making such a wonderful app.

**Reply by bo0tzz** [reply]
No. If a feature is being worked on then there will be an open PR and probably a mention of that in the FR comments. We don't provide timelines on features.

**Reply by lippertmarkus** [reply]
@bo0tzz can you change the category of this discussion to be a feature request instead?

**Reply by lippertmarkus** [reply]
nevermind, #1779 and #1587 are duplicates
